### PR TITLE
Par 4 calendar with left right navigation

### PR DESCRIPTION
**Pull request by pjooklas**

### CODE REVIEW

**Review by maizax — APPROVED** [review]
looks good

File: packages/main-fe/src/app/features/components/calendar/calendar.component.ts [hunk 1]
```diff
@@ -1,21 +1,41 @@
...
   constructor() {}
 
-  ngOnInit(): void {}
+  ngOnInit(): void {
+    this.buttonsEnabled();
+  }
+
+  buttonsEnabled() {
+    this.showDate.valueChanges.subscribe(value => {
+      console.log(this.showDate.value == this.maxDate)
```
Comment (Karalito) [inline packages/main-fe/src/app/features/components/calendar/calendar.component.ts]: Console.log left

File: packages/main-fe/src/app/features/components/calendar/calendar.component.ts [hunk 2]
```diff
@@ -1,21 +1,41 @@
...
+      console.log(this.showDate.value, this.maxDate)
+      this.buttonPreviousDayEnabled = true;
+      this.buttonNextDayEnabled = true;
+      if (this.showDate.value <= this.minDate) {
+        this.buttonPreviousDayEnabled = false;
+        console.log(this.showDate.value, this.minDate)
```
Comment (Karalito) [inline packages/main-fe/src/app/features/components/calendar/calendar.component.ts]: console.log

File: packages/main-fe/src/app/features/components/calendar/calendar.component.ts [hunk 3]
```diff
@@ -21,18 +26,14 @@ export class CalendarComponent implements OnInit {
   }
 
   buttonsEnabled() {
-    this.showDate.valueChanges.subscribe(value => {
-      console.log(this.showDate.value == this.maxDate)
```
Comment (Karalito) [inline packages/main-fe/src/app/features/components/calendar/calendar.component.ts]: console.log

**Review by Karalito — APPROVED** [review]
Everything looks good

**Pull request merged**